packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -60,6 +60,7 @@ export { default as VListChain } from './list/VListChain';
 export { default as setListItemStyle } from './list/setListItemStyle';
 export { getTableFormatInfo } from './table/tableFormatInfo';
 export { saveTableCellMetadata } from './table/tableCellInfo';
+export { default as pasteTable } from './table/pasteTable';
```

Comment:
Is is used anywhere outside this package? If not, no need to export it

Reply:
Is it not standard to export everything?

Reply:
Anything exported from index means it is expected to be used externally. But if some function is only used internally, and we don't expect it is directly used outside roosterjs, we don't need to export it.

Reply:
So do I add @internal to the function?

Reply:
Correct